### ISSUE TITLE
Accessibility - 2.4.1 Bypass Blocks

### ISSUE THREAD

**Issue by phucduong86** [shiffman/shiffman.net#85]
Results for:
https://shiffman.net/books/
https://shiffman.net/a2z/closures/
https://shiffman.net/a2z/cfg/

Findings: Page does not have proper heading structure/landmarks or skip navigation

**Comment by phucduong86** [comment]
Closed - see PR #94
